### ISSUE TITLE
make the name of the specific theming options more clear

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#34511]
We should probably think about renaming these items as their are not really distinct in what they are doing imo:
![image](https://user-images.githubusercontent.com/42591237/194950334-a884ff32-0b6e-46cf-a44f-6df22c6a70fa.png)

_Originally posted by @szaimen in https://github.com/nextcloud/server/issues/34437#issuecomment-1273808059_

cc @skjnldsv

**Comment by szaimen** [comment]
@jancborchardt @nimishavijay what could be good names for them?

**Comment by skjnldsv** [comment]
Can you rephrase the title please?

**Comment by szaimen** [comment]
> Can you rephrase the title please?

done

**Comment by skjnldsv** [comment]
Rough idea maybe?
![image](https://user-images.githubusercontent.com/14975046/195681779-b23e343d-90d1-4f28-b420-fbe87e98a837.png)

**Comment by skjnldsv** [comment]
@szaimen good from me, but the design/ux theme should be the one to say :wink:

**Comment by szaimen** [comment]
cc @jancborchardt and @nimishavijay then ;)

**Comment by nimishavijay** [comment]
Looks great! :) My suggestion:
- "Change color" --> "Solid color"
- I'm also thinking we can move "Remove background" to be the last item, kinda following our pattern of "Delete" always being the last action
- Also one question: is it not possible to upload your own background? 

cc @jancborchardt for a second opinion on the wording

**Comment by szaimen** [comment]
@nimishavijay , did you see my ideas? (Looks like you only saw ths screenshot)...

**Comment by jancborchardt** [comment]
- Agree with @szaimen on "Custom background"
- Default background
- Would slightly adjust @nimishavijay’s proposal to "Plain color" so it succinctly communicates that it’s "only" color
- For "Delete background", an X icon would be better than a trash bin

**Comment by szaimen** [comment]
All right!

**Comment by skjnldsv** [comment]
Thanks for all the feedback here.
So, the `Plain color` is going to be removed, because it is a duplicate between
- Change color
- Default background
- Remove background

Here is what it will look like
![image](https://user-images.githubusercontent.com/14975046/196933431-c54172ba-83cf-4638-82b8-bb76983d5b75.png)

Any thoughts?
What have changed
- The cloud is displayed in the list if the admin changed the default background [image](https://user-images.githubusercontent.com/14975046/196933609-e94e9802-6f6e-4cf4-ac16-8d9ab619fea2.png)
- The border now show the theme colour too
- You can change the colour even if you selected a theme that offers another one [image](https://user-images.githubusercontent.com/14975046/196933930-608db95e-ae97-4898-884a-85f8a5195344.png)
- And obviously, you ca remove the background which show none selected and the colour picker show your current primary colour [image](https://user-images.githubusercontent.com/14975046/196934299-71928159-fa26-42c1-90d1-048caec8373b.png)

**Comment by Jerome-Herbinet** [comment]
Hi @skjnldsv @szaimen @jancborchardt @nimishavijay,

I gave myself time to think after the successful resolution of the folder icon color problem in "Files".

I think the labels still lack clarity and consistency with each other.

**At the moment, we have the following button's labels :** 
- Pick from files
- Default image
- Custom color
- Plain background

**Here is what I think :** 
- "Pick from files" and "Default image" : These two buttons are inconsistent with each other. I think that "Pick from files" should be instead "Custom image". So we would have "Custom image" and "Default image". These two buttons would thus have a common word (Image) and therefore a coherence that would make it clear that both do the same thing, but with a different source. Also, when you read "Pick from files", some might wonder ... picking "what stuff" and for "what usage". "Custom image" is IMO more clear as opposed to "Default image"
- "Custom color" and "Plain background" : As in the previous case, these two buttons are inconsistent with each other. The words used are different while the type of background is the same, but only the source differs. I suggest renaming "Custom color" with "Custom plain background" and "Plain background" with "Default plain background". Also, the wording "" is confusing, because when we talk about "custom color", this notion differs from the notion of background ... and could lead one to think that we are going to apply a custom color to something other than the background. This is not wrong, but the most visible expected change, apart from the change of colors of the folder icons (etc), is the implementation of a solid background color. 

**To summarize, we would have (consistent words, and therefore more understandable) :** 
- Custom background image 
- Default background image
- Custom plain background
- Default plain background

**... instead of :** 
- Pick from files
- Default image
- Custom color
- Plain background

**Simulated visual rendering with my browser's code inspector :** 
![2022-12-14_13-48](https://user-images.githubusercontent.com/33763786/207599453-8e4cd308-e840-4278-aecc-e2f380d50e0f.png)

**Same in French :** 
![2022-12-14_13-56](https://user-images.githubusercontent.com/33763786/207601018-fd9de8af-06e4-4105-a4b0-943cfa9130cb.png)

**Small necessary CSS fix :** 
As the labels would be longer, they would all be displayed on two lines (at least in English but also in French and certainly many other languages); a small CSS property change would have to be made to allow the selection pictogram to be displayed in the right place taking into account a greater text height. Height would be at 60% instead of 100%. See below : 

```.background-selector .background.activenot(.icon-loading)::after {
background-image: var(--original-icon-checkmark-white);
  background-repeat: no-repeat;
  background-position: center;
  background-size: 44px;
  content: "";
  display: block;
  height: 60%; // instead of 100%
}
```

Thanks for reading me.
Best regards.
